### ISSUE TITLE
The results on val datast of ModelNet40 is only about 45.91% !

### ISSUE THREAD

**Issue by auniquesun** [Strawberry-Eat-Mango/PCT_Pytorch#25]
Hi @Strawberry-Eat-Mango  @uyzhang,  Thanks for sharing the PyTorch implementation of PCT!

I follow the instructions in this repo and train PCT from scratch on ModelNet40. First, I prepare the dataset. Then configure the training parameters, shown in Fig.1, and keep others default (6 2080ti devices are used, but it's weird that the last two occupy fewer memories, shown in Fig.2). Third, 'scheduler.step()' was added as other issues suggested to eliminate a warning.

Fig. 1
![1640876090(1)](https://user-images.githubusercontent.com/25259958/147762613-756436a2-7ed8-4f3d-9d02-d52daf3ef611.png)

Fig. 2
![1640876489(1)](https://user-images.githubusercontent.com/25259958/147763089-728e9068-0a4d-4e6a-a1da-7b3ea0ccd9e7.png)

Everything looks good: the loss reduces and the accuracy increases during training. One exceptional point is that the accuracy increases slowly, e.g. Accuracy achieves only 29% after 200 epochs of training, and finally the accuracy stops at 45.91% in 250 epochs of training, shown in Fig.3 . Hence, the results are far from expectations.

Fig. 3
![1640876718(1)](https://user-images.githubusercontent.com/25259958/147763440-39abd69e-95e8-470a-8ba3-426c065d73ee.png)

What are possible reasons? Can anyone give some tips or suggestions?

Appreciate!

**Comment by auniquesun** [comment]
I have addressed this issue!
